pkg/smi/client_test.go

```diff
@@ -261,6 +346,15 @@ var _ = Describe("When listing TrafficTargets", func() {
 
 		_, err := fakeClientSet.smiTrafficTargetClientSet.AccessV1alpha3().TrafficTargets(testNamespaceName).Create(context.TODO(), trafficTarget, metav1.CreateOptions{})
 		Expect(err).ToNot(HaveOccurred())
+
```

Comment:
@michelleN would it be possible to add new tests for these new changes instead of updating the existing ones. I plan on refactoring the existing tests to address issues in the pub-sub usage (already have locally). If it's a separate function, would be much easier for me to incorporate those changes without accidentally leaving them out as a part of the cleanup.

Reply:
sure thing. done.